pnet/Private-Networks-PSK-V1.md

```diff
@@ -0,0 +1,75 @@
+## Pre-shared Key Based Private Networks in IPFS
+
+This document describes the first version of private networks (PN) featured in IPFS.
```

Comment:
Instead of 'first', let's just terminology like 'alpha' that has a clear meaning in software.